### ISSUE TITLE
Generating Octree Leaf size

### ISSUE THREAD

**Issue by Amit10311** [larics/larics_gazebo_worlds#2]
Thank you for this great package for octomap. The simulator is working perfectly. 

Could you please explain more about the normalization transform and leaf size. It gives ocotmap with scale of 0.060767 but I want the ocomap with the same scale of my mesh. 

I look forward to hearing from you. Thank you for your valuable time.

**Comment by aivano** [comment]
@Amit10311 Can you please provide more details on what you are trying to do?

I didn't understand the underlying problem with scale. If i interpreted correctly, your `.stl` or `.dae` file has some size and the octomap is scaled down after using `binvox` and `binvox2bt`?

**Comment by Amit10311** [comment]
@aivano Yes, the octomap is scaled down after using `binvox` and `binvox2bt`.

**Comment by aivano** [comment]
@Amit10311 It happened to me a few times. I think it has something to do with the combination of dimensions used in 3D modeling software and ones used by `binvox`. The workaround I did a was to measure the created octomap and compare it to 3D model measurements, after which I scaled up the 3D model and repeated the procedure.

Unfortunately, I don't know why this happens. If this workaround does not do the job for you, I suggest you send me the link to your 3D model so I can try to see what happens.

**Comment by Amit10311** [comment]
@aivano It's model from company which I can't share. It's making an octomap but there is a gap between two objects. Can you please tell me how to fix this?

**Comment by aivano** [comment]
@Amit10311 You can try to increase the map resolution with `./binvox -e res path_to_file `. Depending on the size of your model you can try with `res=4096`. This could provide a more detailed octomap. If the model is small you can further increase resolution.
